litreview.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+Continuous ATO (C-ATO) is an organizational initiative to automate security compliance activity. In collaboration with industry, NIST is developing the Open Security Controls Assessment Language (OSCAL), a set of formats expressed in XML, JSON, and YAML. These formats provide machine-readable representations of control catalogs, control baselines, system security plans, and assessment plans and results. A security team will likely only work with OSCAL indirectly but through a tool. The purpose of OSCAL is a standardized, machine-readable format to share artifacts between Governance, Risk, and Compliance (GRC) and software automation tools or programs.
+
+---
+
+<div class="usa-alert usa-alert--info">
+  <div class="usa-alert__body">
+    <h4 class="usa-alert__heading">Use Case 1: Leverage Security Artifacts Between Agencies</h4>
+    <p class="usa-alert__text">
+      Agency A uses C-ATO to authorize a new platform. Agency B wants to authorize the same platform and leverage the machine-readable artifacts of Agency A to shorten their authorization process from months to days.
+    </p>
+  </div>
+</div>
+
+---
+
+C-ATO can help streamline the generation and maintenance of A&A artifacts through automation. Currently, these activities are time-consuming and labor-intensive. Steps for C-ATO adoption:
+1. define a strategy
+2. make a plan
+3. ready your organization
```

Comment:
Re these adoption/maturity steps, this means the implied prerequisite environment is green field and an organization that is "starting from scratch" to use a metaphor? It might help to address these assumptions in the beginning of document regarding what is and isn't in scope of the the work.